core/src/main/java/org/eclipse/hono/util/TenantObject.java

```diff
@@ -538,4 +538,14 @@ public static JsonObject newAdapterConfig(final String type, final boolean enabl
                 .put(TenantConstants.FIELD_ADAPTERS_TYPE, type)
                 .put(TenantConstants.FIELD_ENABLED, enabled);
     }
+
+    /**
+     * Gets the concurrent connections limit for a tenant.
+     *
+     * @return The concurrent connections limit.
+     */
+    @JsonIgnore
+    public long getConnectionsLimit() {
+        return json.getLong(TenantConstants.CONNECTIONS_LIMIT, TenantConstants.DEFAULT_CONNECTIONS_LIMIT);
```

Comment:
Done.

Reply:
I think that for consistency reasons, you should not access the json field directly but use `TenantObject.getProperty(String, Object)` so that we can be sure to get the same behavior whenever we retrieve data ...